docs/src/pages/docs/crocks/Tuple.md

```diff
@@ -474,3 +474,40 @@ trimap(add(10), toUpper, add(10), triple)
 ```
 
 </article>
+
+<article id="topic-transformation">
+
+## Transformation Functions
+
+#### tupleToArray
+
+`crocks/Tuple/tupleToArray`
+
+```haskell
+tupleToArray :: Tuple a -> [ a ]
+tupleToArray :: (a -> Tuple b) -> a -> [ b ]
+```
+
+Used to transform a given `Tuple` instance to an `Array` instance. 
+
+Like all `crocks` transformation functions, `tupleToArray` has two possible
+signatures and will behave differently when passed either
+a `Tuple` instance or a function that returns an instance
+of `Tuple`. When passed the instance, a transformed `Array` is
+returned. When passed a `Tuple` returning function, a function will
+be returned that takes a given value and returns an `Array`.
+
+```javascript
+import Tuple from 'crocks/Tuple'
+
+import tupleToArray from 'crocks/Tuple/tupleToArray'
+
+const Triple = Tuple(3)
+
+const triple = Triple( 1, { key: 'value' }, 'string' )
+
+tupleToArray(triple) // => [ 1, { key: 'value' }, 'string' ]
+
+tupleToArray(() => triple)() // => [ 1, { key: 'value' }, 'string' ]
```

Comment:
Not sure i like this as an example as the function used is impure, but it's so simple i might wait for @evilsoft to chime in.

Reply:
perhaps this:
`tupleToArray(identity)(triple) // => [ 1, { key: 'value' }, 'string' ]`